src/meta/MetaServiceUtils.cpp

```diff
@@ -905,26 +905,6 @@ std::string MetaServiceUtils::genTimestampStr() {
     return ch;
 }
 
-folly::Optional<bool> MetaServiceUtils::isIndexRebuilding(kvstore::KVStore* kvstore) {
```

Comment:
Maybe don't move them to `BaseProcessor`? some class doesn't inherit from `BaseProcessor`

Reply:
yeah， good point. will fix.